tests/test_tensor/test_module_spec.py

```diff
@@ -110,28 +122,79 @@ def run_linear_with_spec(label):
     assert tensor_shard_equal(model.weight.grad, model_handy.weight.grad)
     assert tensor_shard_equal(model.bias.grad, model_handy.bias.grad)
 
+def run_check_shared_param():
+    from transformers import BertForMaskedLM, BertConfig
+    hidden_dim = 8
+    num_head = 4
+    sequence_length = 12
+    num_layer = 2
+    vocab_size = 30524
```

Comment:
you can use a smaller V. It is just a unitest.

Reply:
Done